3rdparty/python/BUILD

```diff
@@ -1,8 +1,11 @@
 # Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-# See requirements.txt in this directory to change deps.
-python_requirements()
+python_requirements(
+    overrides={
+        "humbug": {"dependencies": ["#setuptools"]},
```

Comment:
This was legitimate. I'm not sure why Poetry was working..

Reply:
Poetry probably doesn't isolate like Pex does and `pip` / `wheel` / `setuptools` are standard in many venvs?